udfs/community/README.md

```diff
@@ -185,3 +208,119 @@ returns:
 | 5	| 50 | 12.649110640673518 |
 
 
+### [csv_to_struct(strList STRING)](csv_to_struct.sql)
```

Comment:
Please alphabetize the entries in the README and add to the table of contents at the top.

The following are missing from the TOC:
* csv_to_struct
* find_in_set
* get_value
* get_array_value